### PR TITLE
#165413092 Configure Eslint

### PR DESCRIPTION
**Pull request by Kabalisa**
#### Description
for developers to write quality codes they need to follow best practises and conventions in their codes. but because it may be hard to follow them, we need to configure a Linting tool such as eslint that will help developer by showing them what to change in their codes. 
#### Type of change
N/A
#### How Has This Been Tested?
it is tested by running the ```npm run lint``` Script created in package.json 
#### Checklist:
N/A
#### Pivotal Tracker ID
[#165413092](https://www.pivotaltracker.com/story/show/165413092)

### CODE REVIEW

**Review by ericnyirimana — APPROVED** [review]
On my side, everything is fine. You have done what you were typically asked to do in its right context.

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "root": true,
+    "parser": "babel-eslint",
+    "extends": ["airbnb-base", "plugin:prettier/recommended"],
+    "env": {
+      "node": true,
+      "es6": true,
+      "mocha": true
+    },
+    "plugins":["prettier"],
```
Comment (macphilips) [inline .eslintrc.json:10]: is there a specific reason for adding this plugin?

File: package.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint --ignore-path .gitignore ."
```
Comment (macphilips) [inline package.json]: Please update npm scripts to include a lint fix.

```suggestion
    "lint": "eslint --ignore-path .gitignore .",
    "lint:fix": "npm run lint --fix"
```

**Review by macphilips — CHANGES_REQUESTED** [review]
Good job. Please attend to the comment below

File: .eslintrc.json (hunk 1 above)
Comment (Kabalisa) [reply .eslintrc.json:10]: This plugin will help in reducing linting errors and increasing the amount of error that can be fixed automatically with the --fix option.

**Review by Niyitangasam — APPROVED** [review]

**Comment by jabichris** [comment]
Good work

**Review by jabichris — COMMENTED** [review]
Everything is clear.
